test/Concurrency/isolated_parameters.swift

```diff
@@ -154,6 +163,13 @@ func testIsolatedClosureInference(a: A) {
   let _: (isolated A) -> Void = { (a2: isolated A) in
     a2.f()
   }
+
+  let f: (isolated A, isolated A) -> Void = { (a1, a2) in
```

Comment:
worth adding a TODO to emit an warning/error here too if you intend to merge the current state?

Reply:
not merging just yet and i'd like to fix it in this PR.